index.js

```diff
@@ -231,3 +231,8 @@ function noop () {}
 function alwaysTrue () {
   return true
 }
+
+function getHost (addr) {
+  if (addr === '::' || addr === '0.0.0.0') return '127.0.0.1'
```

Comment:
This is kind of what `drives` CLI does, except it returns `localhost` which I think it's better because it means it could be IPv4 or IPv6, but sure it's simpler to return `127.0.0.1` if we want to avoid DNS assumptions as it was mentioned